rosbridge_library/src/rosbridge_library/internal/message_conversion.py

```diff
@@ -127,29 +144,32 @@ def configure(node_handle=None):
             sys.exit(0)
 
 
-def get_encoder():
+def get_encoder() -> Callable[[ListType], bytes]:
```

Comment:
What is the bracket around `ListType` doing?

Reply:
Callable takes 2 type arguments: a list of argument types and the return type. The brackets create a list with a single item